src/site/_includes/components/CodePattern.js

```diff
@@ -1,7 +1,11 @@
-const {html} = require('common-tags');
 const Prism = require('prismjs');
-const patterns = require('../../_data/patterns')();
+const patterns = require('../../_data/patterns').patterns;
 
+/**
+ * A component to display code samples and a code demo side-by-side.
+ * @param {string} patternId Id of the Code Pattern to be displayed.
+ * @param {number} [height=300] Optional desired height of the component.
+ */
 module.exports = (patternId, height = 300) => {
```

Comment:
Just a drive-by, this 300 is what makes our unstyled content work well because we're not reflowing the page. I'm happy with this.